### PR TITLE
Added new Avi module to setup PKI profile.

### PR DESCRIPTION
**Pull request by grastogi23**
##### ISSUE TYPE

 - New Module Pull Request

##### COMPONENT NAME

network.avi.pkiprofile

##### ANSIBLE VERSION

```
ansible 2.3.0
  config file = /home/grastogi/.ansible.cfg
  configured module search path = Default w/o overrides
```

##### SUMMARY

New module to setup PKI profile.

```

```

### CODE REVIEW

**Comment by khaltore** [comment]
shipit

**Comment by ericsysmin** [comment]
shipit

File: lib/ansible/modules/network/avi/avi_pkiprofile.py [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+        description:
+            - When enabled, avi will only validate the revocation status of the leaf certificate using crl.
+            - To enable validation for the entire chain, disable this option and provide all the relevant crls.
+            - Default value when not specified in API or module is interpreted by Avi Controller as True.
+extends_documentation_fragment:
+    - avi
+'''
+
+EXAMPLES = """
+- name: Example Adds/Deletes PKIProfile configuration from Avi Controller
```
Comment (gundalow) [inline lib/ansible/modules/network/avi/avi_pkiprofile.py]: Example appears to add, rather than delete.

**Review by gundalow — CHANGES_REQUESTED** [review]

**Comment by ericsysmin** [comment]
shipit

**Review by gundalow — APPROVED** [review]

**Pull request merged**